crmsh/bootstrap.py

```diff
@@ -1909,21 +1951,6 @@ def update_nodeid(nodeid, node=None):
                                               ringXaddr_res)
                 if not ringXaddr:
                     error("No value for ring{}".format(i))
-
```

Comment:
Is this code executed if unicast is set? I cannot really understand the whole process.
I'm just afraid if this can affect to multicast

Reply:
In fact, why do we have things related to multicast in that method?

Reply:
This block of code is under flag `is_unicast`, if this flag is Flase(which means is in multicast mode), this code wouldn't be executed.
So this block of code has no related with multicast